src/test-kit/OptionButtons/selectors.ts

```diff
@@ -0,0 +1,5 @@
+export const selectors = {
+   sortByButton: 'button:has-text("Sort By")',
```

Comment:
Why not use `getByText`?

Reply:
https://playwright.dev/docs/locators#locate-by-text

Reply:
Correct me if I’m wrong, but I don’t think these are equivalent. One will locate all elements containing the text “Sort By,” while the other will narrow it down to buttons only. Nevertheless, it might be a good idea to use `.getByRole('button', { name: 'Sort By' })`, though I’m not entirely sure how to create a selector for this. What do you think?

Reply:
1. getByRole is fine as well.
2. you don't need a selector for that anymore, this is playwright function to generate a locator in a different way, without a selector, playwright is generating the selector from the `getByRole`.

Reply:
Thanks for the clarification. So, to use `.getByRole`?

Reply:
sure